### DISCUSSION TITLE
Changing a grid resolution for orientation maps

### DISCUSSION THREAD

**Discussion by GazizovMR** [mtex-toolbox/mtex#298] [Ask Anything] [answered]
Dear Ralf Hielscher,
Could you help me... I would like to decrease grid resolution for orientation maps without changing data (smoothing, averaging, etc.).
For example, I have 4x4-mkm OM with resolution of 700x700 pixels. But I would like to get  4x4-mkm OMs having resolutions of 350x350 pixels or 175x175 pixels by deleting each second or third row/column in the grid.
What script can I use for it?
--
Sincerely,
Marat

**Comment by ralfHielscher** [comment, marked as answer]
Have a look at http://mtex-toolbox.github.io/files/doc/EBSD.reduce.html
```matlab
ebsd = ebsd.reduce
```
Ralf.

**Comment by GazizovMR** [comment]
Thank you very much!
